javascript/javascript1/week1/houseyPricey.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Hack Your Future, Javascript, homework Week1
+ * this file includes:
+ 
+  3. Housey pricey (A house price estimator)
+*/
+
+
+// 3. Housey pricey (A house price estimator)
+
+let myFriendsHouses = [
+
+    {
+        friendName:"Peter",
+        houseWidth:8,
+        houseHight:10,
+        houseDepth:10,
+        gardenSizeInM2:100,
+        price:2500000
+    },
+
+    {
+        friendName:"Julia",
+        houseWidth:5,
+        houseHight:8,
+        houseDepth:11,
+        gardenSizeInM2:70,
+        price:1000000
+    }
+];
+
+
+for(let i = 0; i < 2; i++){
+    let volumeInMeters = myFriendsHouses[i].houseHight * myFriendsHouses[i].houseDepth * myFriendsHouses[i].houseWidth;
+    let housePrice = volumeInMeters * 2.5 * 1000 + myFriendsHouses[i].gardenSizeInM2 * 300;
+    console.log("Actual price: " + housePrice);
+    console.log("Paid price: " + myFriendsHouses[i].price);
+
+    if(housePrice >= myFriendsHouses[i].price)
+    {
+        console.log(myFriendsHouses[i].friendName + ", You got the house on a good price.")
+    }else{
+        console.log(myFriendsHouses[i].friendName + ", You paide more than the actual price.");
+    }
+}
+
```

Comment:
1 You can use const instead of let for 'myFriendsHouses'. 
2.  for(let i = 0; i < 2; i++) what happen if I have 10 objects inside 'myFriendsHouses'?
It will print only first two array objects. Fix it, So it will print all, even I have 2 or 10 or 1000.

Reply:
Thanks for the tip. Changes have been done.